pkg/crc/machine/status.go

```diff
@@ -57,13 +64,20 @@ func (client *client) Status() (*types.ClusterStatusResult, error) {
 		DiskUse:   diskUse,
 		DiskSize:  diskSize,
 	}
-	if vm.bundle.IsOpenShift() {
+	switch {
+	case vm.bundle.IsPodman():
+		logging.Info("podman")
+		clusterStatusResult.PodmanVersion = vm.bundle.GetPodmanVersion()
+		clusterStatusResult.Preset = preset.Podman
+	case vm.bundle.IsMicroshift():
+		logging.Info("microshift")
```

Comment:
i think this `log.Info` is unintended and left over from debugging

Reply:
@anjannath ah yes, I will remove those.

Reply:
@anjannath Updated.